src/main/drivers/serial.h

```diff
@@ -88,14 +88,18 @@ typedef struct serialPort_s {
     uint8_t identifier;
 } serialPort_t;
 
+// TODO(hertz@): adjust this to include all the UARTDevice_e elements + necessary SOFTSERIALS
+// # define SERIAL_PORT_MAX_INDEX MAX_UARTDEV
+# define SERIAL_PORT_MAX_INDEX 11
```

Comment:
I need some ideas here.
In my opinion this should be set to max element of UARTDevice_e + 1, e.g. `LPUARTDEV_1 + 1`. Shall I include `drivers/serial_uart.h` here and use this enum-constant directly?